aequilibrae/project/database_specification/transit/tables/stops.sql

```diff
@@ -32,17 +28,14 @@
 CREATE TABLE IF NOT EXISTS stops (
 	stop_id           TEXT     PRIMARY KEY,
 	stop              TEXT     NOT NULL ,
-	agency_id         INTEGER  NOT NULL,
```

Comment:
It is better to keep the agency ID and use the agency ID with the most services going through that stop.